src/KafkaFlow.Retry/Durable/Definitions/Builders/Polling/PollingDefinitionBuilder.cs

```diff
@@ -0,0 +1,20 @@
+﻿namespace KafkaFlow.Retry
+{
+    public abstract class PollingDefinitionBuilder<SELF> where SELF : PollingDefinitionBuilder<SELF>
```

Comment:
Can we re-write `SELF` to `T` or `TPollingDefinitionBuilder`, or another suggestion?

Reply:
`SELF` was an attempt to evidence that is an auto-reference here. I will change to `T` anyway

Reply:
I will change to ``TSelf`